earthfile2llb/parser/EarthLexer.g4

```diff
@@ -36,10 +36,10 @@ END: 'END' -> pushMode(COMMAND_ARGS);
 Command: [A-Z]+ -> pushMode(COMMAND_ARGS);
 
 NL: WS? COMMENT? CRLF;
-WS: [ \t] ([ \t] | LC)*;
+WS: [ \t] ([ \t] | (LC NL*))*;
```

Comment:
If we have a line continuation, we can possibly have many CRLF in sequence, without a need for other thing